src/core/CoolBoard.cpp

```diff
@@ -134,8 +132,8 @@ void CoolBoard::loop() {
       this->sendSavedMessages();
     }
   }
-  SPIFFS.end();
-  if (this->sleepActive && (!this->shouldLog() || !rtcSynced)) {
+  if (this->sleepActive && (!this->shouldLog() || !rtcSynced) ||
+      !this->powerCheck()) {
```

Comment:
No need for that, let it sleep.
Imagine the batterie is above `VOLTAGE_MIN` at the beginning of the loop, but below at the end and consider what this would do:
1. do not sleep for `logInterval` but for 900s
2. Then wake up and sleep for 900s
3. Then wake up again, and sleep again for 900s
4. And so on until the power drains completely or device is plugged in

Versus
1. Sleep for about `logInterval`
2. Then wake up and sleep for 900s
3. Then wake up again, and sleep again for 900s
4. And so on until the power drains completely or device is plugged in

The difference is very slight and not worth the changes in my opinion. However, checking the battery at the **beginning of the loop** seems very important : a lot of filesystem operations occur outside the loop itself so if we only check at the end, damage to the SPIFFS might already have been done and the check is too late...

Remove that but please put the same logic at the end.